### DISCUSSION TITLE
Fosterer weekly email configuration

### DISCUSSION THREAD

**Discussion by inghamn** [sheltermanager/asm3#1473] [Q&A / Help]
With the new version of ASM deployed, staff would like to start using the weekly email to fosterers feature.  We've checked the boxes in Options -> Medical, but the emails are not going out when the cron runs.  I checked the mail logs for Monday as well as Sunday.

Is there other configuration we need to enable in asm.conf?
![screen](https://github.com/sheltermanager/asm3/assets/630282/4baa8e17-865b-4054-9f0d-3029899e59be)

**Comment by bobintetley** [comment]
There shouldn't be anything else you need to do. Is it that there's no medical items due for your fosterers? The logs should show a message if this is the case.

Later versions let you choose the day of the week to send on. I think the default with this version was weekday==0 so Monday. The cron task has to run on a Monday for it to work. This is called as party of the "daily" cron job.

**Comment by inghamn** [comment]
Which log should I look in.  Nothing in Reports->Auditing looks useful.

When I do the SQL queries movement.send_fosterer_emails, I get back data, so I think I am expecting the emails to be sent.

**Reply by bobintetley** [reply]
Assuming you left the asm3.conf logging options as the defaults (log_debug = true and log_location = syslog / local3), then look in /var/log/syslog or /var/log/debug - the attempt to send an email along with any error messages should be reported there.

**Reply by inghamn** [reply]
Ah, that makes sense.  I will enable debug, and check on this again next Monday morning.
